docs/source/user_guide/runtime-conf.md

```diff
@@ -198,9 +199,39 @@ kubectl describe configmap -n kubeflow kfp-tekton-config
 
 Example: `Argo`
 
+#### Apache Airflow settings
+
+This section defines the settings for the Apache Airflow deployment that you want to associate with this runtime configuration.
+
+##### api_endpoint
+
+The Apache Airflow API endpoint you want to utilize. This setting is required.
+
+Airflow Example: `http://your.apache.webserver:port`
```

Comment:
Same here - no need to call out this as special to Airflow.

Reply:
done